index.js

```diff
@@ -0,0 +1,16 @@
+import { initForm } from "./src/scripts/gallery/formImageUploader.js";
+import { initImageDrag, initColumnResize } from "./src/scripts/gallery/dragAndDrop.js";
+import { initConrolButtons } from "./src/scripts/gallery/controlButtons.js";
+import { initGalleryObserver } from "./src/scripts/gallery/infiniteScrollGallery.js";
+
+document.addEventListener("DOMContentLoaded", () => {
```

Comment:
What do you think Would it be better if we moved these initialization functions to the main control function?
```suggestion
function main() {
   ... // Some code 
}

document.addEventListener("DOMContentLoaded", main)
```

Reply:
yes, it will be better, done